sphinxgallery/__init__.py

```diff
@@ -5,4 +5,4 @@
 
 def path_static():
     """Returns path to packaged static files"""
-    return os.path.abspath(os.path.dirname(__file__))+'/_static'
+    return os.path.join(os.path.abspath(os.path.dirname(__file__)), '_static')
```

Comment:
Super minor niggle, it feels more natural to have the abspath as the outer call (i.e. `abspath(join(...))` rather than `join(abspath(...))`) even if the result is the same ...